common/src/test/java/tachyon/util/io/PathUtilsTest.java

```diff
@@ -16,13 +16,41 @@
 package tachyon.util.io;
 
 import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 import tachyon.Constants;
+import tachyon.thrift.InvalidPathException;
 
 public class PathUtilsTest {
+  @Rule
+  public final ExpectedException mException = ExpectedException.none();
+
+  @Test
+  public void getPathComponentsTest() throws InvalidPathException {
+    try {
```

Comment:
NIT: better to split this testcase into two testcases: one is the test with successful results (i.e., Asserts within the try-catch block), and one test is expecting the exception.